pootle/apps/pootle_app/management/commands/flush_cache.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+# This must be run before importing Django.
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand, CommandError
+
+from django_redis import get_redis_connection
+
+from pootle.core.models import Revision
+from pootle_store.models import Unit
+
+
+class Command(BaseCommand):
+    help = """Flush cache."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--django-cache',
+            action='store_true',
+            dest='flush_django_cache',
+            default=False,
+            help='Flush django default cache.',
+        )
+        parser.add_argument(
+            '--rqdata',
+            action='store_true',
+            dest='flush_rqdata',
+            default=False,
+            help=("Flush revision counter and all RQ data (queues, pending or "
+                  "failed jobs, refresh_stats optimisation data). "
+                  "Revision counter is restores automatically. "),
+        )
+        parser.add_argument(
+            '--stats',
+            action='store_true',
+            dest='flush_stats',
+            default=False,
+            help='Flush stats cache data.',
+        )
+        parser.add_argument(
+            '--all',
+            action='store_true',
+            dest='flush_all',
+            default=False,
+            help='Flush all caches data.',
+        )
+
+    def handle(self, **options):
+        if (options['flush_stats'] or options['flush_rqdata']
+            or options['flush_django_cache'] or options['flush_all']):
+            self.stdout.write('Flushing cache...')
+        else:
+            raise CommandError('No options were set.')
```

Comment:
Were provided. But I guess if we can use argparse for this instead of checking this ourselves.